production/helm/promtail/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: promtail
-version: 0.6.3
+version: 0.7.1
```

Comment:
I think you need to bump it again ? @steven-sheehy

Reply:
Yep and loki-stack. Kind of sucks people have to bump and fix conflicts all the time, but it's worth it until a release so that charts that people are using don't get overwritten in [repo](https://github.com/grafana/loki/tree/gh-pages/charts).